src/common/CqlEditorTerminologySection.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import "./CqlEditorTerminologySection.scss";
+
+interface CqlEditorTerminologySectionProps {
+  title: string;
+  children?: any;
+}
+
+const CqlEditorTerminologySection = (
+  props: CqlEditorTerminologySectionProps
+) => {
+  const { title, children } = props;
+  const [open, setOpen] = useState(true);
+  const chevronClass = open ? "chevron-display open" : "chevron-display";
+  const growingDivClass = open ? "growing-div open" : "growing-div";
+
+  return (
+    <div
+      className="cql-editor-tab-heading"
```

Comment:
these IDs/classes I think should be terminology specific and not cql editor

Reply:
Sure, I changed them to terminology-section-tab-heading, also IDs to have terminology-section instead of cql-editor